### DISCUSSION TITLE
build error while excuting yarn build:vscode on riscv

### DISCUSSION THREAD

**Discussion by ShenChongwu** [coder/code-server#6793] [Q&A] [answered]
`root@lpi4a:/home/sipeed/bench/node/code-server# yarn build`
`yarn run v1.22.22`
`$ ./ci/build/build-code-server.sh`
`Done in 143.56s.`
`root@lpi4a:/home/sipeed/bench/node/code-server# export VERSION='0.0.1' && yarn build:vscode`
`yarn run v1.22.22`
`$ ./ci/build/build-vscode.sh`
`error: pathspec 'product.json' did not match any file(s) known to git`
`error Command failed with exit code 1.`
`info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

**Comment by code-asher** [comment, marked as answer]
Ah you might have missed the install and submodule steps.  We list them under the development section but not the build section.  We should probably list them in both sections.

https://github.com/coder/code-server/blob/ab4b36f573c6d336dd79efa7859ed922fa00f81b/docs/CONTRIBUTING.md#L90-L93
